src/components/messenger/list/conversation-item.tsx

```diff
@@ -107,7 +107,9 @@ export class ConversationItem extends React.Component<Properties> {
 
     const { messagePreview, lastMessage } = this.props.conversation;
     if (messagePreview) {
-      return messagePreview;
+      const isUserLastMessageSender = lastMessage?.sender?.userId === this.props.myUserId;
+      const lastSenderDisplayName = isUserLastMessageSender ? 'You' : lastMessage.sender.firstName;
```

Comment:
Just want to double check if you verified this with admin messages. Admin messages have a sender but it's really a fake thing so I don't think we want any "sender" prefix for those ones.

Reply:
Good shout. I've adjusted this.